ext/bg/js/profile-conditions.js

```diff
@@ -36,135 +39,140 @@ function _profileConditionTestDomainList(url, domainList) {
     return false;
 }
 
-const _profileModifierKeys = [
-    {optionValue: 'alt',   name: 'Alt'},
-    {optionValue: 'ctrl',  name: 'Ctrl'},
-    {optionValue: 'shift', name: 'Shift'}
-];
+let profileConditionsDescriptor = null;
```

Comment:
At some point, this file should be refactored into a class. This isn't necessarily due to changes in this commit; there are several scripts which store state and/or have private global functions that should be converted to classes at some point.

This is basically just a heads up as the file grows in complexity.

Reply:
Yeah, I was thinking the same. The environment dependency could be given to the constructor by either `Backend` or settings.